packages/swap/src/routes/__tests__/v2/swap.test.ts

```diff
@@ -1805,86 +1413,7 @@ describe('server', () => {
       const { swapId, ...rest } = body;
       expect(status).toBe(200);
-      expect(rest).toMatchSnapshot({
-        state: 'FAILED',
-        srcAsset: 'ETH',
-        srcChain: 'Ethereum',
-        destAsset: 'DOT',
-        destChain: 'Polkadot',
-        destAddress: '1yMmfLti1k3huRQM2c47WugwonQMqTvQ2GUFxnU7Pcs7xPo',
-        estimatedDurationSeconds: 48,
-        depositChannel: {
-          createdAt: 516000,
-          brokerCommissionBps: 0,
-          depositAddress: '0x6aa69332b63bb5b1d7ca5355387edd5624e181f2',
-          srcChainExpiryBlock: '265',
-          estimatedExpiryTime: 1699527060000,
-          isExpired: false,
-          openedThroughBackend: false,
-          srcChainRequiredBlockConfirmations: 2,
-          fillOrKillParams: {
-            retryDurationBlocks: 15,
-            refundAddress: '0x541f563237a309b3a61e33bdf07a8930bdba8d99',
-            minPrice: '29387358770557187699218413430556141945466.638919302188',
-          },
-          dcaParams: { numberOfChunks: 10, chunkIntervalBlocks: 3 },
-        },
-        deposit: {
-          amount: '5000000000000000000',
-          receivedAt: 552000,
-          receivedBlockIndex: '92-400',
-        },
-        swap: {
-          totalOutputAmountSwapped: '8385809332068',
-          totalInputAmountSwapped: '9999899999999300000',
-          totalChunksExecuted: 2,
-          currentChunk: {
-            swapInputAmount: '4999949999999650000',
-            swapOutputAmount: '4192904666034',
-            scheduledAt: 552000,
-            scheduledBlockIndex: '92-399',
-            executedAt: 564000,
-            executedBlockIndex: '94-594',
-            retryCount: 0,
-            fees: expect.any(Array),
-          },
-          lastExecutedChunk: {
-            swapInputAmount: '4999949999999650000',
-            swapOutputAmount: '4192904666034',
-            scheduledAt: 552000,
-            scheduledBlockIndex: '92-399',
-            executedAt: 564000,
-            executedBlockIndex: '94-594',
-            retryCount: 0,
-            fees: expect.any(Array),
-          },
-          isDcaSwap: true,
-          fees: expect.any(Array),
-        },
-        swapEgress: {
-          amount: '4192707216034',
-          scheduledAt: 564000,
-          scheduledBlockIndex: '94-595',
-          sentAt: 624000,
-          sentAtBlockIndex: '104-7',
-          sentTxRef: '104-2',
-        },
-        refundEgress: {
-          amount: '999844999999160000',
-          scheduledAt: 564000,
-          scheduledBlockIndex: '94-594',
-          failure: {
-            failedAt: 624000,
-            failedAtBlockIndex: '104-1',
-            mode: 'REFUND_OUTPUT_TOO_SMALL',
-            reason: {
-              name: 'ethereumIngressEgress.BelowEgressDustLimit',
-              message: 'The amount is below the minimum egress amount.',
-            },
-          },
-          ignoredAmount: '999844999999160000',
-        },
-      });
+      expect(rest).toMatchSnapshot();
```

Comment:
not a big fan of these empty methods. adding the expected props gives us two layers of protection. Otherwise I feel like you can easily end up updating snapshots without realising it.

Reply:
Yeah it's just a pain to have to double update stuff. Ideally we should all be reviewing snapshot changes in PRs. Otherwise we should just assert that the object strictly equals